### ISSUE TITLE
Mouse position improperly calculated on Windows with fractional DPI scaling

### ISSUE THREAD

**Issue by konistehrad** [thepowerprocess/UnitySpritePivotEditor#3]
Hello!

First: thank you so much for this wonderful tool. It's gonna save me a ton a time. I did find a small issue, though, and that's in how mousePosition is being calculated. Unfortunately, `e.mousePosition` is somewhat unreliable, for both the inverted Y that you've accounted for, but also for fractional scaling. Fortunately, there's a utility that manages this: 

```
        Event e = Event.current;
        Vector2 mousePos = HandleUtility.GUIPointToScreenPixelCoordinate(e.mousePosition);
        Vector3 position = sceneView.camera.ScreenToWorldPoint(mousePos);
```

This should be all that's required to project the mouse position into the "real" position within the scene view window, and quickly fetch the position using ScreenToWorldPoint.

Thanks again, and excellent work!

**Comment by thepowerprocess** [comment]
@konistehrad Thanks for identifying this issue and providing the solution.  I'll update and test it shortly.

**Comment by thepowerprocess** [comment]
fixed in commit 3581c0032c92e5a87d71be072f0536fa354431cf
